### DISCUSSION TITLE
[BUG] SPAM link in plugin list

### DISCUSSION THREAD

**Discussion by jcubic** [facebook/docusaurus#10778] [Q&A]
You should remove the link:

https://docusaurus-powershell.netlify.app/

![](https://slorber-api-screenshot.netlify.app/https%3A%2F%2Fdocusaurus-powershell.netlify.app/showcase)

From the repository of plugins

**Comment by Josh-Cena** [comment]
Which "repository of plugins" are you referring to? I don't see it on our website.

**Reply by jcubic** [reply]
I think that I just assumed that you control this website:

https://docusaurus.community/plugindirectory/

**Reply by Josh-Cena** [reply]
@homotechsual

**Reply by homotechsual** [reply]
Thanks for the tag!

**Reply by homotechsual** [reply]
Fixed - thanks all!

**Reply by jcubic** [reply]
Tagged you on the Open issue on GitHub repo.
